freeglut/freeglut/src/fg_internal.h

```diff
@@ -526,11 +579,14 @@ typedef void (*SFG_Proc)();
  * and for no other reason.  Since it's hidden in the macro, the
  * ugliness is felt to be rather benign.
  */
-#define SET_WCB(window,cbname,func)                            \
+#define SET_WCB(window,cbname,func,udata)                      \
 do                                                             \
 {                                                              \
     if( FETCH_WCB( window, cbname ) != (SFG_Proc)(func) )      \
```

Comment:
Ah. Good eye. I can either add an additional check in the if statement or have a separate else if after that one which only sets user data. Opinions?

Rarely do read a whole API. Certain ones I do... then I keep the header and when a new version comes out, I diff it so I know what changed.